components/basic/MoveToTop.js

```diff
@@ -19,10 +19,15 @@ const ScrollButton = () => {
   };
 
   const scrollToTop = () => {
-    window.scrollTo({
-      top: 0,
-      behavior: "smooth",
-    });
+    const scrollDuration = 500;
```

Comment:
The improvement looks fine to me but do we actually need the custom scroll implementation?